xesar-connect/src/main/kotlin/com/open200/xesar/connect/Topics.kt

```diff
@@ -27,6 +27,105 @@ class Topics(vararg val topics: String) {
+            val CODING_STATION_DELETED = "xs3/1/ces/CodingStationDeleted"
+
+            /** MQTT topic string for the "CalendarDeleted" event. */
+            val CALENDAR_DELETED = "xs3/1/ces/CalendarDeleted"
+
+            /** MQTT topic string for the "AuthorizationTImeProfileDeleted" event. */
+            val AUTHORIZATION_TIME_PROFILE_DELETED = "xs3/1/ces/AuthorizationTimeProfileDeleted"
+
+            /** MQTT topic string for the "AuthorizationProfileDeleted" event. */
+            val AUTHORIZATION_PROFILE_DELETED = "xs3/1/ces/AuthorizationProfileDeleted"
+
+            /** MQTT topic string for the "ZoneCreated" event. */
+            val ZONE_CREATED = "xs3/1/ces/ZoneCreated"
+
+            /** MQTT topic string for the "PersonCreated" event. */
+            val PERSON_CREATED = "xs3/1/ces/PersonCreated"
+
+            /** MQTT topic string for the "OfficeModeTimeProfileCreated" event. */
+            val OFFICE_MODE_TIME_PROFILE_CREATED = "xs3/1/ces/OfficeModeTimeProfileCreated"
+
+            /** MQTT topic string for the "CodingStationCreated" event. */
+            val CODING_STATION_CREATED = "xs3/1/ces/CodingStationCreated"
+
+            /** MQTT topic string for the "CalendarCreated" event. */
+            val CALENDAR_CREATED = "xs3/1/ces/CalendarCreated"
+
+            /** MQTT topic string for the "AuthorizationTimeProfileCreated" event. */
+            val AUTHORIZATION_TIME_PROFILE_CREATED = "xs3/1/ces/AuthorizationTimeProfileCreated"
+
+            /** MQTT topic string for the "UserGroupChanged" event. */
+            val USER_GROUP_CHANGED = "xs3/1/ces/UserGroupChanged"
+
+            /** MQTT topic string for the "ZoneChanged" event. */
+            val ZONE_CHANGED = "xs3/1/ces/ZoneChanged"
+
+            /** MQTT topic string for the "PersonChanged" event. */
+            val PERSON_CHANGED = "xs3/1/ces/PersonChanged"
+
+            /** MQTT topic string for the "OfficeModeTimeProfileChanged" event. */
+            val OFFICE_MODE_TIME_PROFILE_CHANGED = "xs3/1/ces/OfficeModeTimeProfileChanged"
+
+            /** MQTT topic string for the "InstallationPointChanged" event. */
+            val INSTALLATION_POINT_CHANGED = "xs3/1/ces/InstallationPointChanged"
+
+            /** MQTT topic string for the "CodingStationChanged" event. */
+            val CODING_STATION_CHANGED = "xs3/1/ces/CodingStationChanged"
+
+            /** MQTT topic string for the "CalendarChanged" event. */
+            val CALENDAR_CHANGED = "xs3/1/ces/CalendarChanged"
+
+            /** MQTT topic string for the "MediumPersonChanged" event. */
+            val MEDIUM_PERSON_CHANGED = "xs3/1/ces/MediumPersonChanged"
+
+            /** MQTT topic string for the "InstallationPointsInZoneChanged" event. */
+            val INSTALLATION_POINTS_IN_ZONE_CHANGED = "xs3/1/ces/InstallationPointsInZoneChanged"
+            /** MQTT topic string for the "IndividualAuthorizationsAddedToMedium" event. */
```

Comment:
Can you add a blank line, or remove the blank lines between the other topics? How ever you like it more. :)